config/datasetIngest.py

```diff
@@ -10,4 +10,5 @@
 config.dataIngester.load(os.path.join(decamConfigDir, 'ingest.py'))
 config.calibIngester.load(os.path.join(decamConfigDir, 'ingestCalibs.py'))
 config.defectIngester.load(os.path.join(decamConfigDir, 'ingestCalibs.py'))
+config.defectIngester.parse.extnames = []
```

Comment:
Is this supposed to fix the warning that happens on defect ingest about them not being multi-extension FITS files? If so, that's great, but I suggest editing the commit message to clarify this is a general decam defect ingestion bug you're fixing and not an ap_verify specific one.